src/lib/server/editors/json/ConvertToJsonEditor.ts

```diff
@@ -0,0 +1,89 @@
+import type { LLM } from '$lib/server/llms/llm';
+import type { Tables } from '$lib/supabase';
+import { EditorType, RequiredInputType } from '$lib/types';
+import { PromptEditor } from '../editor';
+
+export class ConvertToJSONEditor extends PromptEditor {
+	constructor() {
+		super(
+			'Output JSON',
+			'Output JSON',
+			'Your desired output seems to be very structured. Consider converting it to JSON.',
```

Comment:
Your prompt is requesting structured information from the mode. Consider asking it to return JSON.